src/components/BudgetStatement/BudgetStatementForecast/HeaderWithIcon/HeaderToolTip.stories.tsx

```diff
@@ -0,0 +1,59 @@
+import { useTheme } from '@mui/material';
+import { createThemeModeVariants } from '@ses/core/utils/storybook/factories';
+import HeaderToolTip from './TooltipHeader';
+import type { Meta } from '@storybook/react';
+
+const meta: Meta = {
+  title: 'Fusion/Components/Budget Statements/HeaderToolTip',
+  component: HeaderToolTip,
+
+  parameters: {
+    chromatic: {
+      viewports: [375, 768, 1024, 1280, 1440],
```

Comment:
here it looks like only 375 and 768 have differences, all the other resolutions are the same so maybe we can just let those 2 to save some extra variants in chromatic

Reply:
done